package.json

```diff
@@ -198,7 +198,8 @@
     "requires": [
       "td-agent",
       "cloudteam-td-agent-setup",
-      "dial-agent"
+      "dial-agent",
```

Comment:
Could you raise a separate PR to remove this please. We no longer use dials so should be able to remove this, just do not want to do it in the same PR incase we mask any other failures.

Reply:
alrighto